### PR TITLE
Erp 110/pdf preview

### PR DESCRIPTION
**Pull request by neilk17**

### CODE REVIEW

**Comment by barbinbrad** [comment]
I thought I'd made this comment already, but I don't see it now. Can you remove the `lucide-react` package and just use `react-icons/lu` icons?

File: apps/carbon/app/components/Layout/Navigation/IconSidebar.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ const NavigationIconLink = forwardRef<
     ref
   ) => {
     const iconClasses = [
-      "absolute left-3 top-3 flex rounded-md items-center items-center justify-center", // Layout
+      "absolute left-3 top-3 flex rounded-md items-center items-center justify-center text-primary/80", // Layout
```
Comment (barbinbrad) [inline apps/carbon/app/components/Layout/Navigation/IconSidebar.tsx:112]: This is the only change I don't love. But if you feel strongly about it I'm cool with it.

**Comment by barbinbrad** [comment]
I think we still need to handle non image/pdf files:

https://github.com/barbinbrad/carbon/assets/64510427/7530ec5f-e7d5-4464-ac06-da9ff523d65b

File: apps/carbon/app/components/Layout/Navigation/IconSidebar.tsx (hunk 1 above)
Comment (neilk17) [reply apps/carbon/app/components/Layout/Navigation/IconSidebar.tsx:112]: Yep np, I'll revert this I don't feel strongly about it

**Pull request merged**